.reuse/templates/header.jinja2

```diff
@@ -0,0 +1,11 @@
+This file is part of JavaSMT,
+JavaSMT is an API wrapper for a collection of SMT solvers.:
```

Comment:
These two lines are not a correct sentence, and the punctuation `.:` is also weird. This also affects all changed files.